Tasks/Common/docker-common/dockercommandutils.ts

```diff
@@ -89,6 +91,38 @@ export async function getLayers(connection: ContainerConnection, imageId: string
     return layers.reverse();
 }
 
+export function getImageSize(layers: { [key: string]: string }[]): string {
```

Comment:
Add tests for these methods

Reply:
Have added L0 tests for the two Util functions for image size